packages/hash/integration/src/tests/util.ts

```diff
@@ -90,6 +96,56 @@ import {
   updatePageContents,
 } from "../graphql/queries/page.queries";
 
+const randomStringSuffix = () => {
```

Comment:
This is kind of gross but I was running into "CLASH" errors from Kratos otherwise. I'm still not entirely sure why if I'm honest.

Reply:
At the very least it should give us quite a few test runs before the DB becomes too populated with previous data.

Again, this is a terrible patch and we really need to figure out clean testing environments.

Reply:
cc @TimDiekmann and @thehabbos007 just in case you guys have ideas about cleaner testing environments. (Not asking for a response to this comment thread, just flagging for the next time we discuss it :) )

Reply:
I think this is fine for now. A possibility would be to get around kratos in test-mode, but that's just more code to maintain than just randomly generating these user shortnames.

We could maybe look at allowing test-mode transactions in the Graph API (very basic Start transaction, roll back transaction APIs for testing only). But again, lots of code to maintain to make clean-up easier. I don't have good ideas for better test envs :/